cylc/flow/cycling/iso8601.py

```diff
@@ -895,31 +895,23 @@ def get_dump_format():
 def get_point_relative(offset_string, base_point):
     """Create a point from offset_string applied to base_point."""
     try:
-        if len(re.split('\\+|-', offset_string)) > 2:
```

Comment:
When using regex make sure to use "raw strings" with an `r` before the quote i.e. `r'(\+|-)'` to avoid having to double-escape things